### PR TITLE
Use `SNAME` for `ui` actions in Viewport

### PR DESCRIPTION
**Pull request by KoBeWi**
`is_action_pressed()` methods take StringName arguments, but Viewport calls them with plain Strings. It's only relevant when a Control is focused or when dragging something, but these checks are performed quite often in these situations, so SNAME probably makes sense. I didn't test whether it improves something though.

### CODE REVIEW

File: scene/main/viewport.cpp [hunk 1]
```diff
@@ -2228,51 +2228,51 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
...
 					next = from->_get_focus_neighbor(SIDE_RIGHT);
 				}
 
-				if (p_event->is_action_pressed("ui_down") && input->is_action_just_pressed("ui_down")) {
+				if (p_event->is_action_pressed(SNAME("ui_down")) && input->is_action_just_pressed(SNAME("ui_down"))) {
 					next = from->_get_focus_neighbor(SIDE_BOTTOM);
 				}
 			} else {
-				if (p_event->is_action_pressed("ui_focus_next", true, true)) {
+				if (p_event->is_action_pressed(SNAME("ui_focus_next", true, true))) {
```
Comment (AThousandShips) [inline scene/main/viewport.cpp:2255]:
```suggestion
				if (p_event->is_action_pressed(SNAME("ui_focus_next"), true, true)) {
```

File: scene/main/viewport.cpp [hunk 2]
```diff
@@ -2228,51 +2228,51 @@ void Viewport::_gui_input_event(Ref<InputEvent> p_event) {
...
 					next = from->find_next_valid_focus();
 				}
 
-				if (p_event->is_action_pressed("ui_focus_prev", true, true)) {
+				if (p_event->is_action_pressed(SNAME("ui_focus_prev", true, true))) {
```
Comment (AThousandShips) [inline scene/main/viewport.cpp:2259]:
```suggestion
				if (p_event->is_action_pressed(SNAME("ui_focus_prev"), true, true)) {
```

**Review by AThousandShips — APPROVED** [review]
Makes sense to me, been thinking about the input event names for some time, I think it'd be worth investigating if we should make a `InputStringName` structure for these and check the performance as input events are used in pretty tight places and they are used pretty often

**Comment by akien-mga** [comment]
Thanks!

**Pull request merged**